File: Python/Programs/Mathematics/Factorial/Factorial.py
n = None

while n!=0:
    
    print()
    Num = int(input("Enter a Number (0 to Quit): "))
    n = Num

    Factorial = 1

    if n<=0:
        print("See Ya!")

    else:
        
        while Num!=0:
            Factorial *= Num
            Num -= 1

        print(f"The Factorial of {n} is {Factorial}")

File: C/Recursion/Factorial.c
#include <stdio.h>

int factorial(int x) {
    if (x == 1) {
        return 1;
    }
    return x*factorial(x-1);
}

void main() {
    int n = 30;
    printf("%d! = %d\n", n, factorial(n));
}